paimon-core/src/main/java/org/apache/paimon/table/Table.java

```diff
@@ -164,6 +165,9 @@ default void deleteBranches(String branchNames) {
     @Experimental
     ExpireSnapshots newExpireChangelog();
 
+    /** Expire tags. */
+    TagTimeExpire newExpireTags();
```

Comment:
Dont introduce this method to Table, it is not so useful.

Reply:
addressed. Removed it, just depends on TagAutoManager.